src/sentry/web/frontend/accounts.py

```diff
@@ -121,9 +122,18 @@ def recover_confirm(request, user_id, hash):
 
 
 @login_required
+@require_http_methods(["POST"])
```

Comment:
Why did this change? Was it allowed to be a GET before, or just an oversight from before?